packages/dx-react-scheduler-material-ui/src/templates/day-view/time-panel/cell.jsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import classNames from 'classnames';
+import TableCell from '@material-ui/core/TableCell';
+import { withStyles } from '@material-ui/core/styles';
+import moment from 'moment';
+
+const styles = theme => ({
+  cell: {
+    border: 0,
+    padding: theme.spacing.unit,
+    '&:last-child': {
+      padding: theme.spacing.unit,
+    },
+  },
+  text: {
+    ...theme.typography.caption,
+  },
+});
+
+const CellBase = ({
+  classes,
+  className,
+  time,
+  ...restProps
+}) => {
+  const currentTime = moment(time);
+  return (
+    <TableCell
+      numeric
+      className={classNames(classes.cell, className)}
+      {...restProps}
+    >
+      <span className={classes.text}>
+        {currentTime.format('h:mm A')}
```

Comment:
There isn't a test that checks cell content rendering.

Reply:
done